src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorTplPageCmsMasterPageDefSpot.class.php

```diff
@@ -40,39 +40,30 @@ protected function OnAfterCopy()
     }
 
     /**
+     * @deprecated since 6.3.0 - revision management is no longer supported
```

Comment:
We can instead remove the method - the parent now returns false anyway.

Reply:
removed